src/debuginfo.cpp

```diff
@@ -728,50 +719,44 @@ static void jl_getDylibFunctionInfo(char **name, char **filename, size_t *line,
         if (SymFromAddr(GetCurrentProcess(), dwAddress, &dwDisplacement64,
                         pSymbol)) {
             // SymFromAddr returned success
-            jl_copy_str(name, pSymbol->Name);
-            saddr = (void*)(uintptr_t)pSymbol->Address;
+            if (name)
+                jl_copy_str(name, pSymbol->Name);
+            if (saddr)
+                *saddr = (void*)(uintptr_t)pSymbol->Address;
+            if (symsize)
+                *symsize = pSymbol->Size;
         }
         else {
             // SymFromAddr failed
             //jl_printf(JL_STDERR,"SymFromAddr returned error : %lu\n", GetLastError());
         }
 
-        frame_info_line.SizeOfStruct = sizeof(IMAGEHLP_LINE64);
-        if (SymGetLineFromAddr64(GetCurrentProcess(), dwAddress, &dwDisplacement, &frame_info_line)) {
```

Comment:
we may need this. it's unclear to me whether llvm will manage to fix their dwarf emission or codeview reader first